tools/SolveVolatileAbundances.py

```diff
@@ -268,15 +268,14 @@ def get_total_pressure(pin, fO2_shift, global_d):
 
 #====================================================================
 def atmosphere_mass(pin, fO2_shift, global_d):
-    """Atmospheric mass of volatiles and totals for H, C, and N"""
+    """Atmospheric mass of volatiles and totals for H, C, O"""
```

Comment:
What about N? It's in there.

Reply:
Ah, yes, it is. I meant to add O to this comment (since I added the O calculation to the function) but didn't mean to remove the N from the comment.